src/components/PackageCard/Details.tsx

```diff
@@ -30,13 +36,26 @@ export const Details: FunctionComponent = () => {
     author,
     metadata: { date },
     version,
+    name,
   } = usePackageCard();
 
+  const { data } = useStats();
+  const downloads: number | undefined = data?.packages?.[name]?.downloads?.npm;
+
   const authorName = typeof author === "string" ? author : author.name;
 
   return (
     <>
-      <Detail data-testid={testIds.version} label="Version" value={version} />
+      {downloads !== undefined && downloads >= 10 ? (
```

Comment:
BTW - just wanted to bring attention to this. I suggest here not showing downloads if the number of downloads is very low, since according to NPM, there can be a bit of noise in the signal due to downloads just coming from NPM mirrors, which does not represent actual usage of the library. Open to removing this, though.